app/helpers/catalog_helper.rb

```diff
@@ -91,23 +91,23 @@ def date_list( document )
     dateVal
   end
 
-  def display_access_control_level(document)
-
-    accessGroup = document['read_access_group_ssim'] # "public" > "local" > "dams-curator" == "dams-rci" == default
-    viewAccess = nil
-
-    if accessGroup != nil
-
-      if accessGroup.include?('public')
-        viewAccess = nil
-      elsif accessGroup.include?('local')
-        viewAccess = 'Restricted to UC San Diego use only'
+  def display_access_control_level(document, metadata_colls)
+    access_group = document['read_access_group_ssim'] # "public" > "local" > "dams-curator" == "dams-rci" == default
+    view_access = nil

```

Comment:
It looks like the fallback value here, from the `else` condition, is `Curator Only`. If that's true, could you possibly set the initial value of `view_access` to that instead of `nil`? Then you could drop the `else` entirely.

Somewhat related I'm curious why we use `nil` instead of just `''` for `public`, but if it's working then I suppose it's not work messing with ;)